sources/input_to_text_tree.c

```diff
@@ -21,6 +21,7 @@ int	input_to_text_tree(t_param *param)
 	int		ret;
 
 	s = param->cur_command;
+	tree = ft_tree_make();
```

Comment:
No! Not here! This is a mock function, I need it to work! Please introduce your changes to the version below, beyond MOCK_TANIA flag. When this parsing part will be done (when you'll be getting valid trees for at least all the tests), we'll remove the mocking part and only the lower version will stay.

Reply:
this is my question, how to work with MOCK_TANIA, which main i should to use, how to compile it? make tania?

Reply:
Please do not mark my review comments as resolved, I still need them. I mark them resolved myself when I see with my own eyes that my question is answered or the code is adjusted.